src/ConnectionEnum.java

```diff
@@ -0,0 +1,7 @@
+class ConnectionEnum{
+    public enum connectionType{
+        CONNECTION_TYPE_UNKNOWN,
+        ONE_WAY_CONNECTION,
+        TWO_WAY_CONNECTION
+    }
+}
```

Comment:
```suggestion

    public enum connectionType{
        CONNECTION_TYPE_UNKNOWN,
        ONE_WAY_CONNECTION,
        TWO_WAY_CONNECTION
    }

```

no need create a enum inside a class, you can directly have a enum file